From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 6 Mar 2014 11:36:20 +0000
Subject: automake options

Enable automake warnings.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c2dfc39..dc5e48a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,7 +22,7 @@ AC_CONFIG_AUX_DIR([autoconf])
 AC_CONFIG_MACRO_DIR([autoconf])
 AC_CONFIG_HEADERS([ADOL-C/src/config.h])
 AC_PREFIX_DEFAULT(${HOME}/adolc_base)
-AM_INIT_AUTOMAKE([1.11 dist-bzip2 foreign subdir-objects])
+AM_INIT_AUTOMAKE([1.11 dist-bzip2 foreign subdir-objects -Wall])
 AM_MAINTAINER_MODE([disable])
 AM_SILENT_RULES([yes])
 
