From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 31 May 2020 11:24:59 +0100
Subject: typos

---
 ADOL-C/src/taping.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ADOL-C/src/taping.c b/ADOL-C/src/taping.c
index 3b47709..a23db55 100644
--- a/ADOL-C/src/taping.c
+++ b/ADOL-C/src/taping.c
@@ -243,7 +243,7 @@ void fail( int error ) {
         case ADOLC_EXT_DIFF_NULLPOINTER_ARGUMENT:
             fprintf(DIAG_OUT,
                     "ADOL-C error: Got at least one null pointer as argument to"
-                    " extern differnetiated function!\n");
+                    " extern differentiated function!\n");
             break;
         case ADOLC_EXT_DIFF_WRONG_FUNCTION_INDEX:
             fprintf(DIAG_OUT,
